model/adapter.py

```diff
@@ -51,14 +63,23 @@ def get_existing_metabolite(mnx_id, model, compartment):
     """
     if not mnx_id:
         return
+    response = await query_identifiers([mnx_id], 'mnx', model.notes['namespace'])
+    # TODO: so messy, adjust when fixing #30
```

Comment:
#30 is the issue on aim to not have to do syntactic interpretation of id's like EX_x_c.. But great suggestion, adjusted, cheers!